addons/event/models/event_event.py

```diff
@@ -530,6 +528,23 @@ def write(self, vals):
             self.message_subscribe([vals['organizer_id']])
         return res
 
+    def name_get(self):
+        """Adds ticket seat availability if requested by context."""
+        if not self.env.context.get('name_with_seat_availability'):
+            return super().name_get()
+        res = []
+        for event in self:
+            # event or its tickets are sold out
+            if event.event_registrations_sold_out:
+                availability = _("Sold out")
```

Comment:
Wouldn't it be easier to translate with the whole sentence / context, like
```
name = _("%(event_name)s (Sold out)", ...)
name = _("%(event_name)s (%(seats_available)s seats remaining)", ...)
name = _("%(event_name)s (unlimited seats)", ...)
```

That way when people translate, they have a bit more context.